privaterelay/settings.py

```diff
@@ -929,5 +929,8 @@ def set_index_cache_control_headers(
 # django-ftl settings
 AUTO_RELOAD_BUNDLES = False  # Requires pyinotify
 
+# accounts that should not have abuse metrics
+ALLOWED_ACCOUNTS = ["relay-team+e2e@mozilla.com"]
```

Comment:
Normally we'd load this from the environment, but we can do that when we first need it.